packages/ui-components/src/components/tooltip/tooltip.tsx

```diff
@@ -22,6 +22,10 @@ export class KvTooltip implements ITooltip {
 	@Prop({ reflect: true }) position?: ETooltipPosition;
 	/** @inheritdoc */
 	@Prop({ reflect: false }) options?: Partial<ComputePositionConfig> = DEFAULT_POSITION_CONFIG;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) disabled?: boolean = false;
+	/** @inheritdoc */
+	@Prop({ reflect: false }) contentElement?: HTMLElement = null;
```

Comment:
What is this input used for?

Reply:
ContentElement input is added to enable the parent component to specify the reference of an element since we have a slot being rendered in this component. However, this was added to follow other implementations in the library since this input, beyond being optional, is not needed for the purpose of this PR which is to disable the opening of the tooltip when a certain condition is met by the parent component.